app/models/google/calendar/calendar.rb

```diff
@@ -33,7 +33,7 @@ def service
   def user_events
     page_token = nil
     result = client.execute(:api_method => service.events.list,
-                            :parameters => {'calendarId' => 'primary'})
+                            :parameters => { 'calendarId' => 'primary' })
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.